@AH-64D Apache Longbow/Addons/fza_ah64_ufd/functions/fn_update.sqf

```diff
@@ -78,10 +81,10 @@ do {
 
         for "_i" from 0 to 5 do {
             _heli setUserMFDText [MFD_TEXT_IND_UFDTEXT0 + _i, format["%1|%2|%3",
-                if (count _warnings > _i) then {_warnings # _i} else {"          "},
-                if (count _cautions > _i) then {_cautions # _i} else {"           "},
-                if (count _advisories > _i) then {_advisories # _i} else {"          "}
-            ]];
+                if (count _warnings > _i) then {_warnings # _i} else {"          "},       //10
```

Comment:
The number doesn't matter here, and will lead to inaccuracies as we add / remove more

Reply:
Which number doesn't matter? The number of spaces matters, and keeping track of how many spaces are available for each section does as well.